governance/handler.go

```diff
@@ -54,7 +55,7 @@ var GovernanceItems = map[int]check{
 	params.Ratio:                   {stringT, checkRatio, nil},
 	params.UseGiniCoeff:            {boolT, checkUint64andBool, updateUseGiniCoeff},
 	params.DeferredTxFee:           {boolT, checkUint64andBool, nil},
-	params.MinimumStake:            {stringT, checkBigInt, updateMinimumStakingAmount},
+	params.MinimumStake:            {stringT, checkBigInt, nil},
```

Comment:
Why did you remove updateMinimumStakingAmount? I think this change makes the governance item unavailable.

Reply:
The `updateMinimumStakingAmount` is used to update global minimum staking variable which is not used anymore. We get the minmum staking value from database, so I removed this trigger. FYI, the current set still exists and is updated properly on `UpdateCurrentSet` function.